Makefile

```diff
@@ -103,6 +92,11 @@ COMMON_ARGS += --build-arg=GO_BUILDFLAGS="$(GO_BUILDFLAGS)"
 COMMON_ARGS += --build-arg=GO_LDFLAGS="$(GO_LDFLAGS)"
 COMMON_ARGS += --build-arg=http_proxy=$(http_proxy)
 COMMON_ARGS += --build-arg=https_proxy=$(https_proxy)
+COMMON_ARGS += --build-arg=EMBED_NAME=$(NAME)
```

Comment:
once again nit, but I feel our convention was to use same variable names across Dockerfile/Makefile, so it might make sense to drop `EMBED_` prefix

Reply:
Fixed